src/Controller/TeamController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Routing\Annotation\Route;
+
+class TeamController extends AbstractController
+{
+    /**
+     * @Route("/team", name="team")
+     */
+    public function index(KernelInterface $request)
+    {
+        $a = $request->getProjectDir();
+        return $this->render('team/index.html.twig', [
+            'controller_name' => $a,
+        ]);
+    }
+
+    public function groupByStudents(array $projects)
+    {
+        $data1 = file_get_contents('https://hw1.nfq2019.online/students.json');
```

Comment:
Reikėtų stengtis nekartoto to paties kodo. Programavimo nauda ir yra, kad galima tą patį kodo gabalą iškviesti iš kelių vietų.

Plačiau:
 * https://en.wikipedia.org/wiki/Don%27t_repeat_yourself